lib/hydra/works/models/file.rb

```diff
@@ -0,0 +1,35 @@
+module Hydra::Works
+  class File < Hydra::PCDM::File
+
+    # Need this here until projecthydra-labs/hydra-pcdm#77 is closed
+    metadata do
+      configure type: RDFVocabularies::PCDMTerms.File
```

Comment:
Can we update ActiveTriples to support multiple types. That would greatly simplify this code.